### PR TITLE
Feature/healthcheck read credentials

### PR DESCRIPTION
**Pull request by galiri**

### CODE REVIEW

File: config.json.template [hunk 1]
```diff
@@ -79,7 +80,7 @@
       ]
    }
 ],
-"splunk-config": {
+"splunkConfig": {
```
Comment (khatton-ft) [inline config.json.template]: This doesn't match the AppConfig struct any more, so one needs to change. (This change does make it consistent with other properties so I'm generally in favour).

File: healthcheck.go [hunk 2]
```diff
@@ -337,6 +366,38 @@ func (h *readEnvironmentHealthcheck) checkReadEnvironmentReachable() (string, er
...
 
+func (h *readEnvironmentHealthcheck) checkReadEnvironmentCredentials() (string, error) {
+
+	var endpointURL *url.URL
+	for _, metric := range appConfig.MetricConf {
+		var err error
+		if metric.Alias == "content" {
+			endpointURL, err = url.Parse(h.env.ReadUrl + metric.Endpoint)
+			if err != nil {
+				errorLogger.Printf("Read env credentials healthcheck cannot parse url [%v], Err: [%v]", metric.Endpoint, err.Error())
```
Comment (khatton-ft) [inline healthcheck.go]: I think you should `return nil, err` after this log, otherwise the code will panic once the loop has exited.

**Comment by izzyblues** [comment]
I did not go through all your changes, but maybe it makes sense to write few unit/integration tests. Do you agree?
[Here](https://github.com/Financial-Times/methode-content-placeholder-mapper/blob/master/resources/healthcheck_test.go#L31) there is an example on how to mock a HTTP server.

**Comment by tamas-molnar** [comment]
stale

**Pull request closed without merging**